apps/shop/models.py

```diff
@@ -138,3 +156,24 @@ class OrderModel(models.Model):
 
     def __str__(self):
         return f'OrderModel {self.id} by {self.first_name} {self.last_name}'
+
+
+class Review(models.Model):
+    STATE_CHOCES = [
+        ('pending', 'в очікуванні'),
+        ('visible', 'видимий'),
+        ('invisible', 'невидимий')
+    ]
+    item_set = models.ForeignKey(ItemModel, related_name='review_set', on_delete=models.CASCADE)
+    email = models.EmailField(max_length=255)
+    username = models.CharField(max_length=30)
```

Comment:
Change username field to first_name and last_name
Before making migrations delete 2 last migrations